client/src/pages/automation/project/utils/saveWorkflowDefinition.ts

```diff
@@ -46,15 +46,15 @@ export default async function saveWorkflowDefinition(
         parameters,
         type: `${componentName}/v1/${actionName || newNodeComponentDefinition.actions?.[0].name}`,
     };
+    const workflowDefinition: WorkflowDefinitionType = JSON.parse(workflow.definition!);
```

Comment:
prazan red

Reply:
To je Code editor za Script komponentu. Tebi prepuštam smanjivanje :). Preimenovati ću Connections u ConnectionsPanel